controllers/articles.controllers.js

```diff
@@ -30,6 +30,14 @@ exports.patchArticleById = (req, res, next) => {
   const { article_id } = req.params;
   const { inc_votes } = req.body;
 
+  if (isNaN(Number(article_id))) {
```

Comment:
You don't need to check these manually. If they are not the correct data type, you will get a PSQL error anyway :)